src/domains/ecommerce-common/03_storage.tf

```
@@ -378,138 +283,23 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_transactions_c
       OpCountForQueue("PutMessage", queueKey)
       | join OpCountForQueue("DeleteMessage", queueKey) on count_
       | project name = queueKey, Count = count_ - count_1
+      | where toint(Count) > ${each.value.threshold}
     };
     MessageRateForQueue("%s")
     QUERY
-    , format("%s/pagopa-%s-weu-%s", module.ecommerce_storage_transient.name, var.env_short, "ecommerce-transactions-close-payment-queue")
+    , "${module.ecommerce_storage_transient.name}/pagopa-${var.env_short}-weu-${each.value.queue_key}"
```

If you remove the `ecommerce` prefix from the props above you should be able to refactor this to:

```suggestion
    , "${module.ecommerce_storage_transient.name}/pagopa-${local.project}-${each.value.queue_key}"
```